app/src/test/java/net/squanchy/service/firestore/FirestoreFixtures.kt

```diff
@@ -0,0 +1,122 @@
+package net.squanchy.service.firestore
+
+import com.google.firebase.firestore.GeoPoint
+import net.squanchy.service.firestore.model.conferenceinfo.FirestoreConferenceInfo
+import net.squanchy.service.firestore.model.conferenceinfo.FirestoreVenue
+import net.squanchy.service.firestore.model.schedule.FirestoreDay
+import net.squanchy.service.firestore.model.schedule.FirestoreEvent
+import net.squanchy.service.firestore.model.schedule.FirestorePlace
+import net.squanchy.service.firestore.model.schedule.FirestoreSchedulePage
+import net.squanchy.service.firestore.model.schedule.FirestoreSpeaker
+import net.squanchy.service.firestore.model.schedule.FirestoreTrack
+import java.util.Date
+
+fun fixtureOfFirestoreTrack(
+    id: String = "fakeId",
+    name: String = "fakeTrack",
+    accentColor: String? = "#ABCDEF",
+    textColor: String? = "#FEDCBA",
+    iconUrl: String? = "www.squanchy.net"
+) = FirestoreTrack().apply {
+    this.id = id
+    this.name = name
+    this.accentColor = accentColor
+    this.textColor = textColor
+    this.iconUrl = iconUrl
+}
+
+fun fixtureOfFirestoreSpeaker(
+    id: String = "speakerId",
+    name: String = "Boaty McBoatface",
+    bio: String = "Hello, I'm a fake",
+    companyName: String? = "Squanchy",
+    companyUrl: String? = "www.squanchy.net",
+    personalUrl: String? = "www.squanchy.net",
+    photoUrl: String? = "@DonuldTrump",
+    twitterUsername: String? = null
+) = FirestoreSpeaker().apply {
+    this.id = id
+    this.name = name
+    this.bio = bio
+    this.companyName = companyName
+    this.companyUrl = companyUrl
+    this.personalUrl = personalUrl
+    this.photoUrl = photoUrl
+    this.twitterUsername = twitterUsername
+}
+
+fun fixtureOfFirestorePlace(
+    id: String = "placeId",
+    name: String = "Turin",
+    floor: String? = "This floor"
+) = FirestorePlace().apply {
+    this.id = id
+    this.name = name
+    this.floor = floor
+}
+
+fun fixtureOfFirestoreDay(
+    id: String = "today",
+    date: Date = Date()
```

Comment:
⚠️ You should use a fixed value! Leaving it like this may make the tests hit edge cases on some machines. Make it work with the defaults you specify in the `FirestoreEvent` down below (as in, they would be on the same day)